crates/holochain_cascade/src/lib.rs

```diff
@@ -56,15 +56,15 @@ use std::sync::Arc;
 use std::time::Instant;
 use tracing::*;
 
-#[cfg(feature = "test_utils")]
-use kitsune_p2p::dependencies::kitsune_p2p_types::{box_fut_plain, tx_utils::ShareOpen};
-
 pub mod authority;
 pub mod error;
 
 mod agent_activity;
 mod metrics;
 
+#[cfg(feature = "test_utils")]
+use futures::FutureExt;
```

Comment:
Not asking for a change here, just making suggestions

With an import that's used inside a single function, you can put the `use` statement at the top of the function.
The other option, I recently learned, is to do do `Box::pin(async move { Ok(result) })` so that you don't need the `.boxed` that this trait is providing.

Reply:
Okay, I'm happy to change that :+1: